CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Contribution guide
+
+This guide aims to help developers working on this repository.
+
+## Changing all images without copy-paste
+
+We keep the multiple files for the different image versions in sync. Here is a
+tip on how to use `git` and `sed` to automatically apply patches targeting one
+image to all the others, without incurring in manual copy-paste.
+
+Supposing that changes were made to files in the 3.2 directory, we can then
+apply those changes to all other images:
+
+```bash
+for version in 2.4 2.6 3.0-upg; do
+  git diff -- 3.2 | sed "s|3.2/|${version}/|g" | git apply
```

Comment:
If it doesn't apply I get an error from `git apply` and I'm fine with that.

How does running `patch -p1 --dry-run` and `git-apply --check` improve this work flow?
@php-coder @praiskup do you actually run those commands with those flags?

I see `--reject` could be useful to partially apply patches.